usaspending_api/conftest_helpers.py

```diff
@@ -9,20 +9,26 @@
 from usaspending_api.common.helpers.sql_helpers import ordered_dictionary_fetcher
 from usaspending_api.common.helpers.text_helpers import generate_random_string
 from usaspending_api.etl.es_etl_helpers import create_aliases
-from usaspending_api.etl.management.commands.es_configure import retrieve_transaction_index_template
+from usaspending_api.etl.management.commands.es_configure import (
+    retrieve_transaction_index_template,
+    retrieve_award_index_template,
+)
 
 
 class TestElasticSearchIndex:
-    def __init__(self):
+    def __init__(self, *args):
```

Comment:
There doesn't appear to be a good reason for dynamic input args. If only `type` is passed in, might as well be explicitly defined.